modules/local/bowtie2_assembly_build.nf

```diff
@@ -0,0 +1,28 @@
+// https://github.dev/nf-core/mag
+process BOWTIE2_ASSEMBLY_BUILD {
+    tag "${meta.assembler}-${meta.id}"
+    label 'process_medium'
+
+    conda (params.enable_conda ? 'bioconda::bowtie2=2.4.2' : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bowtie2:2.4.2--py38h1c8e9b9_1' :
+        'quay.io/biocontainers/bowtie2:2.4.2--py38h1c8e9b9_1' }"
+
+    input:
+    tuple val(meta), path(assembly)
+
+    output:
+    tuple val(meta), path(assembly), path('bt2_index_base*'), emit: assembly_index
+    path "versions.yml"                                     , emit: versions
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    mkdir bowtie
```

Comment:
I'm assuming this is fine since it came from nf-core/mag but this feels weird to me. it feels irrelevant? where is it used?

Reply:
it is used in the`mapping_depth.nf` subworkflow, which takes in the assembly first and builds the index, and then maps with the `align` module, and sorts with samtools, and then I pass to the metabat2/jgisummarizecontigs to get the depth table. It's not super obvious because in the `main.nf` workflow it called the `mapping_depth.nf` subworkflow, which is where the above is used.